src/styles/app.css

```diff
@@ -0,0 +1,12 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+.nasa-logo {
+  margin: auto;
+  width: 500px;
+  height: 300px;
+}
+
```

Comment:
You never need more than 1 consecutive clear line in your code, one of these should be removed